src/toil/wdl/wdltoil.py

```diff
@@ -3497,6 +3525,20 @@ def coerce(self: SelfType, desired_type: Optional[WDL.Type.Base] = None) -> WDL.
         WDL.Value.Base.coerce = old_base_coerce  # type: ignore[method-assign]
         WDL.Value.String.coerce = old_str_coerce  # type: ignore[method-assign]
 
+
+def make_root_job(target: Union[WDL.Tree.Workflow, WDL.Tree.Task], inputs: WDLBindings, inputs_search_path: List[str], toil: Toil, wdl_options: Optional[Dict[str, str]], options: Namespace) -> WDLSectionJob:
+    if options.run_imports_on_workers:
+        # Run WDL imports on a worker instead
+        root_job: WDLSectionJob = WDLImportJob(target, inputs, inputs_search_path, skip_remote=options.reference_inputs, disk_size=options.import_workers_disk, wdl_options=wdl_options)
+    else:
+        # Run WDL imports on leader
+        # Import any files in the bindings
+        input_bindings = import_files(inputs, target.name, toil._jobStore, inputs_search_path, skip_remote=options.reference_inputs)
+        # Run the workflow and get its outputs namespaced with the workflow name.
+        root_job = WDLRootJob(target, input_bindings, wdl_options=wdl_options)
```

Comment:
We probably need to rename `WDLRootJob` since it isn't always the root anymore.

Reply:
Renamed to `WDLStartJob`